bluepysnap/config.py

```diff
@@ -79,6 +82,9 @@ def _resolve_manifest(manifest, configdir):
             if not v.startswith('/'):
                 raise BluepySnapError("{} cannot be resolved as an abs path.".format(k))
 
+        assert '${configdir}' not in result
```

Comment:
I am not sure what you mean here. 
'${configdir}' can be none here because if you use a dict instead of a file you cannot define a configdir. Otherwise it must exists because this is the input path of the config (if it is a file)

Reply:
ok nevermind